pkg/controller/direct/directbase/operations.go

```diff
@@ -0,0 +1,45 @@
+package directbase
+
+import "k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+
+type UpdateOperation struct {
+	object *unstructured.Unstructured
+}
+
+func NewUpdateOperation(object *unstructured.Unstructured) *UpdateOperation {
+	return &UpdateOperation{
+		object: object,
+	}
+}
+
+func (o *UpdateOperation) UnstructuredObject() *unstructured.Unstructured {
```

Comment:
My first read from the fn name  is that I thought you mean the `unstructured.Unstructured` `Object`. https://github.com/kubernetes/apimachinery/blob/v0.31.0/pkg/apis/meta/v1/unstructured/unstructured.go#L45C2-L45C8 
Can we rename this something like `Unstructured()` or `GetUnstructured()`?

Reply:
Good call, done!